givitsite/gatherer/agora.py

```diff
@@ -0,0 +1,137 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+import urllib.request
+import urllib
+from bs4 import BeautifulSoup
+import re
+import csv
+from datetime import datetime
+from friendreq.models import ItemRequest
+
+
+def givit_main():
+    iseek_dict = {'20016': 'מיטה', '20008': 'כיסא', '20009': 'ארון',
+                  '20017': 'ספה', '10029': 'מכונת כביסה', '10006': 'מקרר'}
+    region_dict = {'Tel Aviv': 'תל אביב-יפו', 'Jerusalem': 'ירושלים',
+                   'Beer Sheva': 'באר שבע', 'Haifa': 'חיפה', 'Modiin': 'מודיעין', 'Hasharon': 'הוד השרון'}
+
+    items_list = get_items_list()
+    i = 0
+    for i in range(len(items_list)):
+        iseek = items_list[i][2]
+        my_soup = Agora_Getrequest(iseek)
+        url_list = find_furniture(
+            my_soup, region_dict.get(items_list[i][3]), iseek_dict.get(items_list[i][2]))
+        for item in range(len(url_list)):
+            items_list[i].append(url_list[item])
+        url_list = []
+    return items_list
+
+
+def get_items_list():
+    """
+    this function open our wishlist and return item_list such that every item in this list is a list with all the
+    information about.
+    Returns:
+    item_list (list of lists): list of all items in our wishlist.
+    """
+    wish_list = ItemRequest.objects.all()
+    items_list = [list(item.values()) for item in wish_list.values()]
```

Comment:
because i already start to work on this project in the past my approach was to use the things that i already implement and do only necessary changes just to make sure that everything is work mainly all the workflow, this is definitely need to improve, i will add it as an issue no.